src/commands.rs

```diff
@@ -919,24 +919,24 @@ struct GitCloneArgs {
 /// Push to a Git remote
 ///
 /// By default, pushes any branches pointing to `@`, or `@-` if no branches
-/// point to `@`. Use `--branch` to push a specific branch. Use `--all` to push
-/// all branches. Use `--change` to generate a branch name based on a specific
-/// commit's change ID.
+/// point to `@`. Use `--branch` to push specific branches. Use `--all` to push
+/// all branches. Use `--change` to generate branch names based on specific
+/// commit's change IDs.
 #[derive(clap::Args, Clone, Debug)]
 #[command(group(ArgGroup::new("what").args(&["branch", "all", "change"])))]
 struct GitPushArgs {
     /// The remote to push to (only named remotes are supported)
     #[arg(long, default_value = "origin")]
     remote: String,
-    /// Push only this branch
-    #[arg(long)]
-    branch: Option<String>,
+    /// Push only this branch (can be repeated)
+    #[arg(long, short)]
+    branch: Vec<String>,
     /// Push all branches
     #[arg(long)]
     all: bool,
     /// Push this commit by creating a branch based on its change ID
-    #[arg(long)]
-    change: Option<RevisionArg>,
+    #[arg(long, short)]
+    change: Vec<RevisionArg>,
```

Comment:
I'm not against this, but I'm not sure if `--change` needs a short form.

Reply:
I don't use it much, so I'm not entirely sure. If it's repeatable (for consistency), it's more convenient to use the short option when you repeat it a bunch.

Reply:
In that case, I would leave `--change` without a short form until someone complains about that.

I think `-b` for `--branch` is more common in VCS world, but `--change` here is very specific to `jj git push`, and I'm afraid that we might want to allocate `-c` for another option.

Reply:
👍